message MatrixMsg
{
	required string msgType = 1;
	optional string extraInfo = 2;
	optional int64 count = 3;
	repeated string tasks = 4;
}